### ISSUE TITLE
Stop telling users to e-mail me

### ISSUE THREAD

**Issue by ForbesLindesay** [npm/npm#9948]
Like the vast majority of package maintainers, my preferred method for receiving bug reports is on GitHub.  Apart from anything, most people who go to GitHub realise that someone else has already had their problem and solved it.  Unfortunatley npm keeps telling them to email me:

https://github.com/npm/npm/blob/2724e30b71ac79f5c6e1aa821474c70ab4f0a222/lib/utils/error-handler.js#L203-L204

Could we direct them to the relevant github issues instead?

**Comment by Stephn-R** [comment]
I dont immediately see how this is even feasible. How would you go about building it?

**Comment by aredridel** [comment]
Ooh, if there's package data in the package.json, that could be a lot better!

**Comment by KenanY** [comment]
Something like this?:

``` diff
$ git diff
diff --git a/lib/utils/error-handler.js b/lib/utils/error-handler.js
index cd61f79..a8364b4 100644
--- a/lib/utils/error-handler.js
+++ b/lib/utils/error-handler.js
@@ -200,8 +200,8 @@ function errorHandler (er) {
         'not with npm itself.',
         'Tell the author that this fails on your system:',
         '    ' + er.script,
-        'You can get their info via:',
-        '    npm owner ls ' + er.pkgname,
+        'You can report this error here:',
+        '    npm bugs ' + er.pkgname,
         'There is likely additional logging output above.'
       ].join('\n')
     )
```

**Comment by Stephn-R** [comment]
I like that idea @KenanY , and then just make sure to provide a `CONTRIBUTING.md` file

**Comment by sinkingshriek** [comment]
I really like how `Atom` handles this. I believe they just check the error msg/status(?)  and also checks if it's already been posted in github and allows you to jump to the posted issue. Not sure if it's quite feasible in `npm` but that would definitely bring down number of tickets in issue tracker too.

**Comment by ForbesLindesay** [comment]
It should be possible for cases where the project is hosted in GitHub (and has the repository field configured correctly).

**Comment by juliangruber** [comment]
@trodrigues's change above looks good to me

**Comment by othiym23** [comment]
This is fixed in `npm@3` by b25efc88067c843ffdda86ea0f50f95d136a638e, which somebody will need to backport to `npm@2` if they want it there. Thanks to @trodrigues for the fix!

**Comment by trodrigues** [comment]
Also added it for `npm@2` at #10628
